src/operator/tensor/la_op.cc

```diff
@@ -187,6 +187,7 @@ NNVM_REGISTER_OP(_backward_linalg_gemm2)
 
 NNVM_REGISTER_OP(_linalg_potrf)
 .add_alias("linalg_potrf")
+.add_alias("_npi_cholesky")
```

Comment:
You may need to register `_npi_cholesky` separately since `np.linalgcholesky` does not allow `a.ndim <= 2`. You will need to perform the error handling in a new shape inference function.